risc0/zkp/src/prove/accum.rs

```diff
@@ -88,7 +95,9 @@ impl<'a, F: Field> CircuitStepHandler<F::Elem> for Handler<'a, F> {
     fn call(
         &mut self,
         cycle: usize,
+        // TODO what is name?
         name: &str,
+        // TODO what is extra?
```

Comment:
This is an extra string argument that is only used by the `log` extern call. 

The extern call interface isn't particularly clean or well formed, but it's complicated by having to deal with calling thru C++ to rust and is generated by zirgen.